pkg/prometheus/statefulset.go

```diff
@@ -628,12 +628,23 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *Config, ruleConfigMapName
 				fmt.Sprintf("--webhook-url=%s", localReloadURL),
 			},
 			VolumeMounts: []v1.VolumeMount{},
-			Resources: v1.ResourceRequirements{
+			Resources:    v1.ResourceRequirements{Limits: v1.ResourceList{}},
+		}
+
+		if c.ConfigReloaderCPU != "0" {
+			container.Resources.Limits[v1.ResourceCPU] = resource.MustParse(c.ConfigReloaderCPU)
+		}
+		if c.ConfigReloaderMemory != "0" {
+			container.Resources.Limits[v1.ResourceMemory] = resource.MustParse(c.ConfigReloaderMemory)
+		}
+
+		if c.ConfigReloaderCPU != "0" && c.ConfigReloaderMemory != "0" {
```

Comment:
we can safely eliminate this block now, right?

Reply:
:facepalm:

Reply:
fixed